R/flex_dashboard.R

```diff
@@ -181,17 +178,7 @@ flex_dashboard <- function(fig_width = 6.0,
 
   # force to fill it's container (unless the option is already set)
   if (is.na(getOption('DT.fillContainer', NA))) {
-    options(DT.fillContainer = TRUE)
-    exit_actions <- c(exit_actions, function() {
-      options(DT.fillContainer = NULL)
-    })
-  }
-
-  # request that DT auto-hide navigation (unless the option is already set)
-  if (is.na(getOption('DT.autoHideNavigation', NA))) {
-    exit_actions <- c(exit_actions, function() {
-      options(DT.autoHideNavigation = NULL)
-    })
```

Comment:
Appears this is no longer a datatables option -- https://datatables.net/reference/option/

Also appears JJ forgot to set this option anyway in this commit to restore options on exit https://github.com/rstudio/flexdashboard/commit/854a2ceb867522fa892d66417a43733e476c92ec